pybamm/models/submodels/thermal/base_thermal.py

```diff
@@ -249,6 +258,12 @@ def _get_standard_coupled_variables(self, variables):
                 # Current collector
                 "Negative current collector Ohmic heating [W.m-3]": Q_ohm_s_cn,
                 "Positive current collector Ohmic heating [W.m-3]": Q_ohm_s_cp,
+                # Lumped cooling
```

Comment:
Can calculating and adding these variables be a separate method that for now only gets called by the lumped model? Other models handle cooling differently and use different parameters so right now this will either not work as the parameters aren’t there or will work but give a result that may be different from averaging the actual cooling in the model.

Reply:
I moved it to lumped model. Is there an easy equivalent for x-full or should we leave it to only work with the lumped model?

Reply:
You’ll have to sum the cooling terms over the boundaries, but you’ll have to do all those calculations manually because of how the model is set up. It assumes a pouch and cooling from the two large faces normal to the through cell direction show up as boundary conditions either end. The other cooking all shows up as a source term (start from 3D and integrate over y,z so the boundary terms become sources). 

Probably not worth doing unless someone asks for it. The x-full model isn’t really that useful since it’s just for a single layer.